src/Components/AddBook.js

```diff
@@ -49,6 +51,24 @@ export default function AddBookForm() {
             className="userInput Title"
             onChange={handleChange}
```

Comment:
- [ ] It would be better if we can run the application without any errors, however, the browser errors not allowing the app to run. Here is the screenshot of the errors in the browser
![Screenshot from 2022-09-22 18-20-54](https://user-images.githubusercontent.com/46388113/191822915-cf9f473b-d6f6-4d24-9f79-fe04c8dbef5c.png)

### NB: 
The application compiled successfully